src/components/ImageView/index.tsx

```diff
@@ -6,31 +7,51 @@ import styles from './styles.module.scss';
 type PublicProps = {
   _btoa?: typeof btoa;
   _log?: typeof log;
+  _sanitize?: typeof purify.sanitize;
   mimeType: string;
   content: string;
 };
 
 type ConvertImageDataParams = {
   _btoa?: typeof btoa;
   _log?: typeof log;
+  _sanitize?: typeof purify.sanitize;
   content: string;
+  mimeType: string;
 };
 
-export const convertImageData = ({
+const convertImageData = ({
   _btoa = btoa,
   _log = log,
+  _sanitize = purify.sanitize,
   content,
+  mimeType,
 }: ConvertImageDataParams): string | null => {
   try {
-    return _btoa(content);
+    return _btoa(
+      // Run sanitize on svg files.
+      mimeType === 'image/svg+xml' ? _sanitize(content) : content,
```

Comment:
Another bit of paranoia 😛 : what about converting `mimeType` to lower case before making the comparison? I think an upper case `IMAGE/SVG+XML` would still be valid in the browser (if someone figured out how to do that).